runtime-modules/project-token/src/tests/fixtures.rs

```diff
@@ -761,7 +762,7 @@ impl IssueRevenueSplitFixture {
             start: None,
             duration: BlockNumber::from(DEFAULT_SPLIT_DURATION),
             allocation_source: member!(1).1,
-            allocation: Balance::from(DEFAULT_SPLIT_ALLOCATION),
+            allocation: Balance::from(DEFAULT_SPLIT_REVENUE),
```

Comment:
The allocation/revenue naming is still inconsistent here (the field name is `allocation`, but you're assigning `revenue`).
This is also the case in some tests (`burn_fails_with_active_revenue_split`, `issue_revenue_split_ok_with_allocation_leftovers_retained_by_issuer`)